utils/cascoda-sdk/Makefile

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2020 Cascoda Ltd.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cascoda-sdk
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Cascoda/cascoda-sdk.git
+PKG_SOURCE_DATE:=2020-02-04
+PKG_SOURCE_VERSION:=745711e26b31c87d707ffc8054a94e3086be08f5
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_VERSION).tar.gz
```

Comment:
Three of these are default and not needed. MIRROR_HASH would need updating as default archive is .xz.